mysql-test/suite/innodb/r/innodb.result

```diff
@@ -2550,7 +2550,7 @@ drop table t2,t1;
 set foreign_key_checks=0;
 create table t2 (a varchar(10), foreign key (a) references t1(a)) engine = innodb DEFAULT CHARSET=latin1;
 create table t1(a varchar(10) primary key) engine = innodb DEFAULT CHARSET=latin1;
-alter table t1 convert to character set utf8;
+alter table t1 convert to character set utf8, algorithm=copy;
```

Comment:
Same here: please do not remove the variant that does not use explicit `algorithm=copy`. It is OK to additionally test with that.